controllers/flowcollector_controller.go

```diff
@@ -18,11 +22,21 @@ import (
 // Make sure it always matches config/default/kustomization.yaml:namespace
 // See also https://github.com/operator-framework/operator-lib/issues/74
 const operatorNamespace = "network-observability"
+const finalizerName = "flows.netobserv.io/finalizer"
 
 // FlowCollectorReconciler reconciles a FlowCollector object
 type FlowCollectorReconciler struct {
 	client.Client
-	Scheme *runtime.Scheme
+	Scheme      *runtime.Scheme
+	flowsConfig ovs.FlowsConfigController
```

Comment:
Maybe this can be renamed, something like `ovsConfigController` or `ovsFlowsConfigController` ? The `ovs` prefix to avoid confusion with the flow collector CR / controller and `Controller` since it's actually not a config object, also to avoid confusion with the other `ovs.flowsConfig` struct.